src/utils/responseHandlers.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from "express";
+import { logger } from "../services";
+import { STATUS_CODES } from "../utils";
+import { customError } from "../error";
+
+export const sendResponse = <T>(res: Response, statusCode: number, data: T) => {
+  return res.status(statusCode).json({
+    data,
+  });
```

Comment:
The spec does not have a consistent response model. This is good.
```suggestion
  return res.status(statusCode).json({
    message: "Data retrieved successfully",
    data,
  });
```